lib/cmdln.py

```diff
@@ -237,13 +240,13 @@ class MyCmd(cmdln.Cmdln):
         if self.optparser: # i.e. optparser=None means don't process for opts
             try:
                 self.options, args = self.optparser.parse_args(argv[1:])
-            except CmdlnUserError, ex:
+            except CmdlnUserError as ex:
```

Comment:
Hrm, this breaks python 2.4 and 2.5 compat. I'm not sure how important that is. Definitely means a major rev bump.